mypy/messages.py

```diff
@@ -294,7 +294,7 @@ def format_bare(self, typ: Type, verbosity: int = 0) -> str:
                         arg_strings.append(
                             self.format_bare(
                                 arg_type,
-                                verbosity = max(verbosity - 1, 0)))
+                                verbosity=max(verbosity - 1, 0)))
```

Comment:
this was the only thing keeping back E251 -- since this isn't a function definition I went ahead and fixed it

Reply:
The comment for ignoring E251 indicates that this is against our style, so perhaps this should be left as-is?

Reply:
I think the comment was about spaces in typed function definitions (`def f(x : int = 0)`); I don't think mypy's style requires parentheses around `=` in calls.

Reply:
Ah that is definitely possible.

Reply:
and pycodestyle fixed the false-positive there pretty quickly :) (it used to get angry about type-annotated default-arg functions)